src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,72 @@
 'use strict';
 
+const NAME = '';
```

Comment:
The constant NAME is declared but never used. Consider removing it if it's not intended for future use.